### DISCUSSION TITLE
Is it possible for FastF1 for something like this?

### DISCUSSION THREAD

**Discussion by siddharth-tewari** [theOehrly/Fast-F1#96] [Q&A]
@theOehrly Hello, I make Youtube videos of Lap by Lap positions of races. I take the data from ergast. 
Link - 
<a href="http://ergast.com/api/f1/2021/21/laps?limit=1000&offset=0">http://ergast.com/api/f1/2021/21/laps?limit=1000&offset=0</a>
<a href="https://youtu.be/oBwat_aOlaA">https://youtu.be/oBwat_aOlaA</a>

Is it possible to convert this data into excel file.
![Screenshot 2022-01-18 135153](https://user-images.githubusercontent.com/76590952/149898817-7f0107f8-6112-4349-9231-9cfe44d15094.jpg)
I made this manually.

**Comment by siddharth-tewari** [comment]
Also can you help in the code to get the Session object and download the api file from it directly then as you suggested once. (TeamRadio.json)

**Comment by theOehrly** [comment]
You can load the data in json format from ergast. Then I'd recommend creating a pandas DataFrame from it.
Pandas supports exporting of DataFrames to Excel, as far as I know. But I've never used it

**Reply by siddharth-tewari** [reply]
I tried but it doesn't work as required due to the extra data such as lap time, name of document, limit etc.

**Reply by theOehrly** [reply]
Yes, it does work. Of course, you need to do some post-processing. I see that you are fairly new to programming with Python. But to learn, you need to try. Sometimes for a long time. I certainly don't have time to write your scripts. I'm sorry.

**Reply by siddharth-tewari** [reply]
@theOehrly No problem. I will focus on it later as learning flask.

**Comment by siddharth-tewari** [comment]
@theOehrly I had a thought that can we calculate G-Force for a driver by taking data from telemetry every 0.5 seconds?

**Reply by theOehrly** [reply]
You can get an approximation of the g-force values. Keep in mind though that peak g-forces usually only occur for very short amounts of time. Therefore, you could only calculate g-force of long sweeping corners and similar accurately.
